css/styles.css

```diff
@@ -0,0 +1,409 @@
+    display: none;
+}
+
+.banners__slides {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-wrap: wrap;
+        flex-wrap: wrap;
+    -webkit-box-pack: justify;
+        -ms-flex-pack: justify;
+            justify-content: space-between;
+    row-gap: 1rem;
+    -webkit-column-gap: 1rem;
+       -moz-column-gap: 1rem;
+            column-gap: 1rem;
+}
+
+.slide {
+    padding: 0;
+    margin: 0;
+    position: relative;
+    height: 23.7rem;
+    text-align: left;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    -ms-flex-preferred-size: 28%;
+        flex-basis: 28%;
+    -ms-flex-negative: 1;
+        flex-shrink: 1;
+}
+
+.slide:nth-child(2) , .slide:nth-child(5) {
+    -webkit-box-flex: 1;
+        -ms-flex-positive: 1;
+            flex-grow: 1;
+}
+
+.slide-1 {
+    background-image: url(../img/slide-1-bg.jpeg);
+}
+
+.slide-2 {
+    background-image: url(../img/slide-2-bg.jpeg);
+}
+
+.slide-3 {
+    background-image: url(../img/slide-3-bg.jpeg);
+}
+
+.slide-4 {
+    background-image: url(../img/slide-4-bg.jpeg);
+}
+
+.slide-5 {
+    background-image: url(../img/slide-5-bg.jpeg);
+}
+
+.slide-6 {
+    background-image: url(../img/slide-6-bg.jpeg);
+}
+
+.slide__text {
+    position: absolute;
+    font-size: 3.2rem;
+    font-weight: 700;
+    text-transform: uppercase;
+    color: #ffffff;
+    left: 3rem;
+    bottom: 2rem;
+}
+
+.btn-submit {
+    font-family: 'Lato', sans-serif;
+    font-size: 1.6rem;
+    font-weight: 400;
+    color: #ffffff;
+    text-transform: uppercase;
+    padding: 2.4rem 13.7rem;
+    background-color: #3C3C3C;
+    cursor: pointer;
+    border: none;
+    margin-bottom: 1.6rem;
+}
+
+
+.footer {
+    -webkit-box-sizing: border-box;
+            box-sizing: border-box;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: center;
+        -ms-flex-pack: center;
+            justify-content: center;
+    -webkit-box-align: center;
+        -ms-flex-align: center;
+            align-items: center;
+    height: 10rem;
+}
+
+.footer__site-icon {
+    width: 3.5rem;
+    height: 3.7rem;
+    fill: #3C3C3C;
+}
+
+.footer__copyrite {
+    font-family: 'Lato', sans-serif;
+    font-size: 1.4rem;
+    font-weight: 400;
+    margin-left: 30px;
+}
+
+@media screen and (max-width:1024px) {
+    html {
+        font-size: 8px;
+    }
+
+    .wrapper {
+        max-width: 80%;
+    }
+
+    .section__content {
+        max-width: 80%;
+    }
+}
+
+@media screen and (max-width:768px) {
+    html {
+        font-size: 7px;
+    }
+
+    .slide {
+        -ms-flex-preferred-size: 45%;
+            flex-basis: 45%;
+            -webkit-box-flex: 1;
+        -ms-flex-positive: 1;
+            flex-grow: 1;
+            -ms-flex-negative: 1;
+        flex-shrink: 1;
+    }
+
+    .btn-submit {
+        padding: 2.4rem 8.7rem;
+    }
+}
+
+@media screen and (max-width:320px) {
```

Comment:
Devices less than 320px no need to be supported in responsive. This rule can be removed. Or change to `min-width: 320px`